src/frontend/delete-github-subscription.ts

```diff
@@ -1,16 +1,14 @@
 import { Subscription } from "../models";
 import { Request, Response } from "express";
 
-/**
- * Handle the when a user adds a repo to this installation
- */
 export default async (req: Request, res: Response): Promise<void> => {
-	if (!req.session.githubToken) {
+	const { github, githubToken, jiraHost } = res.locals;
+	if (!githubToken) {
```

Comment:
Can it be that wen the user first opens /jira/configuration page and wants to remove an org, we will return 401 because they don't have GitHub token yet?

Reply:
`/github/subscription` is called from github configuration page: https://github.com/atlassian/github-for-jira/blob/main/static/js/github-configuration.js#L20

Since you're on the github configuration page, you will have a github token by then.